freak/post_processing_freak.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="post_freak" name="Freak Post Processing" version="@VERSION@.0" profile="17.09">
+
+    <description>Post Processing for Freak Output</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    @VERSION_CMD@
+
+    <expand macro="requirements"/>
+
+    <command detect_errors="exit_code"><![CDATA[
+    python3 ${__tool_directory__}/post_processing_freak.py "$freak_data" "$ref" "$sample_id" -d -o "$output";
+
+    #if $name_output.specify_name
+    ln -s "$output" "$name_output.output"
+    #end if
+
+    ]]></command>
+
+    <inputs>
+        <param type="data" name="freak_data" format="txt,tsv,tabular" label="Freak Output" help=""/>
+        <param type="data" name="ref" format="txt,tsv,tabular" label="CNA Reference" help=""/>
```

Comment:
What is this? What does `CNA Reference` refer to and why is it needed?

Reply:
Ah, `CNA Reference` was a typo - should be `Variant Reference`, but that's the labkey table provided as the source of truth for duplicate removal.

Reply:
Ah right, forgot about that little annoyance. Please label as "GeneTrails Variants File" to be consistent with the preprocessing step. Name can be anything but would suggest something like `variants`. The term ref might be confused with the reference variant allele otherwise.